docs/index.md

```diff
@@ -51,27 +54,30 @@ Correctly renders *most* things in-game and does not crash the emulator when end
 #### GLideN64 Public Release 4.0 `OpenGL`{ .badge data-md-color-primary="cyan" } { #glide4 data-toc-label='GLideN64 4.0' }
 
 The most accurate and best looking plugin, but usually crashes the emulator when ending emulation.
+![resolution](/assets/images/glide4settings.png){ align=right }
+
+When configuring the plugin, set `Full screen resolution` to your monitor's native resolution, and aspect ratio to 4:3.
 
 #### Jabo’s Direct3D8 1.6 `DirectX`{ .badge data-md-color-primary="cyan" } { #jabos3d8 data-toc-label='Jabo’s Direct3D8 1.6' }
-Older plugin that uses less resources than GLideN64, but has visual glitches (does not show invincibility). Only use if you can’t run GLideN64 or OpenGL.
+Older plugin that uses less resources than GLideN64, but has visual glitches, such as not showing invincibility. Only use if you can’t run GLideN64 or OpenGL.
 
 
 ### Audio
 
 #### Azimer's Audio v0.30 (Old Driver) `Recommended`{ .badge data-md-color-primary="green" } { #azimers data-toc-label="Azimer's v0.30" }
 
-Standard audio plugin. Other audio plugins may cause desyncs.
+Standard audio plugin.
 
 ![audio plugin configuration](/assets/images/azimers.png){ .azimer-buffer-size align=right }
 
 It is recommended to reduce `Buffer Size` to the 2nd tick (see image) so game audio is less delayed. This is **not saved** and must be done every time the game is launched.
 
 #### No Sound { #nosound data-toc-label="No Sound" }
 
-Simple audio plugin that disables sound entirely. Does not seem to cause desyncs.
+Simple audio plugin that disables sound entirely.
 
 
-## Configure your Controller
+## 5. Configure your Controller
```

Comment:
We could make the controller section a sub-heading of step 4, since it is also a plugin. 

> Additionally the way it's laid it out -- step 4 is kind of useless because the default graphics/audio plugins should be no config.

They should check the graphics plugin to make sure the fullscreen resolution matches their monitor resolution, and they need to know about the audio plugin settings